### PR TITLE
Feature/oscar2

### LINKED ISSUE #17: Upgrade to Oscar 2.0

### PR DESCRIPTION
**Pull request by sasha0**
Closes #17

### CODE REVIEW

File: oscar_elasticsearch/search/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-default_app_config = "oscar_elasticsearch.search.config.SearchConfig"
```
Comment (specialunderwear) [inline oscar_elasticsearch/search/__init__.py:1]: To keep 1.6 compatibility, this need to remain unchanged. Instead, we can specify in the readme to intall the search app from the appcinfig directly:

"oscar_elasticsearch.search.apps. OscarElasticSearchConfig"

File: setup.py [hunk 2]
```diff
@@ -26,8 +26,8 @@
     # specify dependencies
     install_requires=[
         "setuptools",
-        "django-oscar<2.0",
```
Comment (specialunderwear) [inline setup.py:29]: We want to keep oscar 1.6 compatibility if possible, so lets keep this unspecified.

**Review by specialunderwear — CHANGES_REQUESTED** [review]
We want to try and make this oscar 1.6 and oscar 2.0 compatible at the same time, that means the 2.0 appconfig must be ADDED in a separate file. I discussed with @solarissmoke and we think this should be possible, please see my comments

**Comment by sasha0** [comment]
> We want to try and make this oscar 1.6 and oscar 2.0 compatible at the same time, that means the 2.0 appconfig must be ADDED in a separate file. I discussed with @solarissmoke and we think this should be possible, please see my comments

How'd that be possible considering breaking changes in Oscar 2.0? Why won't we cut new version of the package with the Oscar 2.0 support?

File: oscar_elasticsearch/search/__init__.py (hunk 1 above)
Comment (sasha0) [reply oscar_elasticsearch/search/__init__.py]: Ok done.

File: setup.py (hunk 2 above)
Comment (sasha0) [reply setup.py:29]: Done.

**Review by specialunderwear — APPROVED** [review]

**Pull request merged**